radixdlt-core/radixdlt/src/main/java/com/radixdlt/middleware2/network/GetVerticesRequestMessage.java

```diff
@@ -85,13 +87,11 @@
 	@DsonOutput(Output.ALL)
 	private final int count;
 
-	GetVerticesRequestMessage() {
-		// Serializer only
-		this.vertexId = null;
-		this.count = 0;
-	}
-
-	GetVerticesRequestMessage(HashCode vertexId, int count) {
+	@JsonCreator
+	public GetVerticesRequestMessage(
+		@JsonProperty(value = "vertexId", required = true) HashCode vertexId,
+		@JsonProperty(value = "count", required = true) int count
+	) {
 		this.vertexId = Objects.requireNonNull(vertexId);
 		this.count = count;
```

Comment:
We need to check count is valid here, that was one of the problems that @LukasGasior1 found. @LukasGasior1 should it be greater than zero or it can be zero too?

Reply:
Given that, I believe we need to add the same validation in `GetVerticesRequest`